components/src/lib/Modal/Modal.tsx

```diff
@@ -0,0 +1,28 @@
+import * as Styled from "./style";
+
+import BackDrop from "./BackDrop/BackDrop";
+import Container from "./Container/Container";
+import Header from "./Header/Header";
+import Title from "./Title/Title";
+import ButtonContainer from "./ButtonContainer/ButtonContainer";
+import Button from "./Button/Button";
+import CloseButton from "./CloseButton/CloseButton";
+
+export interface ModalProps {
+  isOpen: boolean;
+  children: JSX.Element;
+}
+
+const Modal = ({ isOpen, children }: ModalProps) => {
+  return <Styled.Modal $isOpen={isOpen}>{children}</Styled.Modal>;
+};
```

Comment:
저는 isOpen 관련해서 dipslay:none, block 처리까지 안가도 요렇게 조건부 렌더링을 라이브러리 컴포넌트 내부에서 처리해주면 되지 않을까 생각했습니다ㅎㅎ

```suggestion
const Modal = ({ isOpen, children }: ModalProps) => {
  return {isOpen ? <Styled.Modal>{children}</Styled.Modal> : null };
};
```

Reply:
오 해당 방법은 전혀 생각해보지 못했었네요. 조건 연산자를 return 마다 적용해 줄 생각은 전혀 못했었습니다. 늘 전체 불러 오는 곳에서만 해당 부분을 적용하다보니 이러한 접근에 대해서 의식하지 못했었네요! 좋은 피드백 정말 감사합니다!!